test/basic_test.go

```diff
@@ -639,6 +639,7 @@ func TestSimultaneousRequests(t *testing.T) {
 	var wg sync.WaitGroup
 	for i := 0; i < 50; i++ {
 		wg.Add(1)
+		//lint:ignore SA2002 t.Fatalf in go routine will happen only if test fails
```

Comment:
Why not add to global ignore?

Reply:
global ignore file seem to not be used anymore (actually should remove from repo, forgot to do that).

Reply:
updated travis and removed the file